### PR TITLE
[v1 API Docs] Add v1/bulk/observations/series/linked

### PR DESCRIPTION
**Pull request by juliawu**
Add the documentation page for /v1/bulk/observations/series/linked

### CODE REVIEW

File: api/rest/v1/bulk_observations_series_linked.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+nav_order: 4.5
+parent: v1 REST
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/observations/series/linked
+---
+
+# /v1/bulk/observations/series/linked
+
+Returns [observations](/glossary.html#observation) of a specific [variable](/glossary.html#variable) for entities linked to a parent entity by the same property.
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_series_linked.md:13]: s/a specific/multiple

File: api/rest/v1/bulk_observations_series_linked.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+### Query Parameters
+
+| Name                                               | Type | Description               |
+| -------------------------------------------------- | ---- | ------------------------- |
+| key <br /> <required-tag>Required</required-tag>   | string | Your API key. See the [page on authentication](/api/rest/v1/getting_started#authentication) for a demo key, as well as instructions on how to get your own key. |
+| linked_entity <br /> <required-tag>Required</required-tag> | list | [DCIDs](/glossary.html#dcid) of the parent place to query.|
+| entity_type <br /> <required-tag>Required</required-tag> | string | Type of place to query for (e.g. city, county, state, etc.). For a list of available values, see the [Graph Browser page on Place](https://datacommons.org/browser/Place). |
+| variables <br /> <requried-tag>Required</required-tag> | list | [DCIDs](/glossary.html#dcid) of the [variables](/glossary.html#variables) to query. |
+| linked_property <br /> <required-tag>Required</required-tag> | string | [DCID](/glossary.html#dcid) of the property to query. Must be `containedInPlace`.|
+{: .doc-table }
```
Comment (shifucun) [inline api/rest/v1/bulk_observations_series_linked.md:74]: There is also a boolean field "all_facets"

Comment (juliawu) [reply api/rest/v1/bulk_observations_series_linked.md:75]: added!

File: api/rest/v1/bulk_observations_series_linked.md (hunk 1 above)
Comment (juliawu) [reply api/rest/v1/bulk_observations_series_linked.md]: done!

**Review by shifucun — APPROVED** [review]

**Pull request merged**